examples/test_examples.sh

```diff
@@ -4,11 +4,26 @@
 # These examples should be convenient for users to reference, not exercise all our features.
 
 set -u -e -o pipefail
-cd $(dirname $0)
+
+TESTS_ROOT_DIR=$(cd $(dirname "$0"); pwd)
+NODEJS_ROOT_DIR=$(cd $TESTS_ROOT_DIR/..; pwd)
+
+# Check environment
+unameOut="$(uname -s)"
+case "${unameOut}" in
```

Comment:
nit: could be simpler, we may never care about the non-linux case

Reply:
👍